test/mock.js

```diff
@@ -269,6 +269,6 @@ exports.override = function (uri, payload) {
 
 exports.clear = function (uri) {
 
-    Wreck.get = internals.nipple.get;
-    Wreck.post = internals.nipple.post;
+    Wreck.get = internals.wreck.get;
+    Wreck.post = internals.wreck.post;
 };
```

Comment:
EOL exists, ya... wanted to see if travis failed too.  Something is broke since last publish of bell